cmd/wallet-web/src/components/Vaults/DeleteVaultModal.vue

```diff
@@ -0,0 +1,115 @@
+<!--
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+<template>
+  <modal :show="showModal">
+    <template #content>
+      <!--content-->
```

Comment:
Maybe we just keep these comments in the Modal itself and remove from here, the template name is self-explanatory